.github/workflows/python-poetry-release.yaml

```diff
@@ -18,20 +18,23 @@ on:
         default: "3.10"
         type: string
       poetry-version:
-        description: "The Python version for setting up Poetry. (Default is 1.1.12)"
+        description: "The Python version for setting up Poetry. (Default is 1.5.1)"
         required: false
-        default: "1.1.12"
+        default: "1.5.1"
         type: string
-      publish-to-test:
-        description: "If set to false, the package is published to PyPI. (Default is true)"
-        required: false
-        default: true
-        type: boolean
       working-directory:
         description: "The working directory of your Python package. (Default is root directory)"
         required: false
         default: "./"
         type: string
+      changelog:
+        description: "Create changelog for release"
```

Comment:
nitpick: we should either terminate all descriptions with `.` or leave it out everywhere, it should follow a consistent format